### DISCUSSION TITLE
Is there a way to handle user browser back operation after error?

### DISCUSSION THREAD

**Discussion by StupidQu** [remix-run/react-router#12357] [Q&A]
Sorry for asking such a stupid question, but i have searched the internet and found nothing.

My app is like:

```jsx
<BrowserRouter>
  <ErrorBoundary
    FallbackComponent={({ error, resetErrorBoundary }) => (
      <ErrorPage
        resetErrorBoundary={resetErrorBoundary}
        message={error.message}
      />
    )}
    onReset={() => window.history.back()}
  >
    <App />
  </ErrorBoundary>
</BrowserRouter>
```

And when i have an error, the error page is displayed. Now i want to use the browser back operation(sorry, i don't know what's the name of this ![](https://img-cdn.zshfoj.com/153bd680714f2b52f435d958529337cfd1982503dd939c9ee6b172d4349d227e.png)), the URL is changed, but the page won't refresh(it is still the error page).

So, is there a way to reload the page when i click on the "back" button?

**Comment by sergiodxa** [comment]
You can use the `useNavigate` hook to access the `navigate` function, then do `navigate(-1)` to go back in the history stack.
